Docs 8010 edit api groups pa (#255)
mulesoft/docs-api-manager: modules/ROOT/pages/api-groups-manage-api-group-instance-contracts.adoc

= Manage API Group Instance Contracts
ifndef::env-site,env-github[]
include::_attributes.adoc[]
endif::[]

> After you grant access to API Group instances in Exchange, a contract is automatically created for it. You can then manage these contracts from API Manager.

Reviewer: ```suggestion
After you grant access to API Group instances in Exchange, a contract is automatically created for that access. You can then manage that contract from API Manager.
```

Reviewer: Clarify: does the contract cover the user's access to ONE API instance or to ONE API group or ONE API group instance?  = 1 API, 1 group, or 1 instance of a group?

Author: Done! Thanks!

> The API Instance contract details are displayed on the API instance page at two levels: all contracts that are directly with the API instance (API instance contracts) and all contracts made with the API groups that consists of this API instance.

Reviewer: API group instance contract? (API Group Instance contract?)
(I think all these "group" and "instance" s/b lowercase, but whatever. What does the contract cover: one API, the group of APIs, or  one instance of the group of APIs?)

Reviewer: Guessing:
```suggestion
The details of a contract for one API instance are displayed in two places:

* With those of any other contracts for that same API instance
* With those of any other contracts in the group that contains that API instance
```
You might want to illustrate this at some point, because the names for these things is confusing. Are we talking about both an API instance contract and a contract for one instance in an API group?

Author: Done! Thanks!

> An application can have more than one contract with the different APIs in an API Group. When you view the API instance contracts at the API Group-level, the contracts are sorted by the applications that are in contract with the API Group. The contract that is currently active within an API Group is displayed as `Applied`.

Reviewer: Is this intended to be the first mention of an APPLICATION having the contract?

Reviewer: Yeah...I really need to see this. 
Just be sure that the text is clear about what two elements have a contract together, and how many types of contract there are.

Author: Ok np! Done!

.The screen capture displays the contract levels and the contract status for the API instance contracts.
image:api-groups-instance-contracts.png[]
<1> The contracts for an API instance at the instance-level
<2> The API Group for which the contracts are displayed
<3> The applications that currently have a contract with the API instance
<4> The contract that is currently active
<5> The contracts for an API instance at the API Group-level

To view API Group instance contracts:

. In API Manager, click *API Groups*.
. From the API Group instance details page, click the `version:instance` link for the API Group instance.
+
The API Group instance details page is displayed.
. From the left navigation, click *Contracts* and expand the instance for which you want to view the contract information:
+
image:api-group-instance-contracts.png[View Contracts for API Group Instance]
+
The contracts for the API Group instances are displayed.

> To manage API Group instance contracts:

Reviewer: manage or revoke?
The following task steps are exactly the same as for viewing, with the additional step of revoking.

Author: Yes, leaving as is. Done! Thanks!
